hs_core/tasks.py

```diff
@@ -106,9 +116,9 @@ def setup_periodic_tasks(sender, **kwargs):
         sender.add_periodic_task(crontab(minute=15, hour=0, day_of_week=1, day_of_month='1-7'),
                                  send_over_quota_emails.s())
         sender.add_periodic_task(crontab(minute=00, hour=12), daily_odm2_sync.s())
-        sender.add_periodic_task(crontab(day_of_month=1), monthly_group_membership_requests_cleanup.s())
+        sender.add_periodic_task(crontab(minute=15, hour=1, day_of_month=1), monthly_group_membership_requests_cleanup.s())
```

Comment:
this was actually the reason that the failing job appeared to be retrying every minute -- because the CRONTAB request didn't specify a single minute, so it was executing every minute of the first day of the month!
https://docs.celeryq.dev/en/v4.4.7/userguide/periodic-tasks.html#crontab-schedules